src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -0,0 +1,40 @@
+package frc.robot.subsystems;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.DoubleSolenoid.Value;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+
+public class Climber extends SubsystemBase{
+    
+    public static DoubleSolenoid doubleSolenoid;
+    public static TalonSRX leaderTalon, followerTalon;
+
+    public Climber() {
+        doubleSolenoid = new DoubleSolenoid(Constants.ClimberDoubleSolenoied1, Constants.ClimberDoubleSolenoied2);
+        leaderTalon = new TalonSRX(Constants.ClimberTalonSRX1);
+        followerTalon = new TalonSRX(Constants.ClimberTalonSRX2);
+        followerTalon.follow(leaderTalon);
+    }
+
+    /*
+    Changing talons speed by demand.    
+    */
+    public void setLifterSpeed(double demand) {
```

Comment:
`demand` has nothing to do with anything. This method sets the speed (percentage of [-1, 1]) to the motor.
```suggestion
    public void setLifterSpeed(double speed) {
```